src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Carriers/Blocks/information_block.html.twig

```diff
@@ -0,0 +1,35 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% trans_default_domain 'Admin.Shipping.Help' %}
+
+{% if showHeaderAlert %}
+  <div class="alert alert-info">
+    <p>{{ 'Your online store needs to have a proper carrier registered in PrestaShop as soon as you start shipping your products. This means sending yours parcels using your local postal service, or having a contract with a private carrier which in turn will ship your parcels to your customers. In order to have PrestaShop suggest the most adequate carrier to your customers during their order checkout process, you need to register all the carriers with which you have chosen to work.'|trans }}</p><br>
+    <p>{{ 'PrestaShop comes with a number of carrier modules that you can activate. You can also buy carrier modules on the PrestaShop Addons marketplace. Recommended modules are listed below: install the module that matches your carrier, and configure it!'|trans }}</p><br>
+    <p>{{ 'If there is no existing module for your carrier, then you can register that carrier by hand using the information that it can provide you: shipping rates, regional zones, size and weight limits, etc. Click on the "%add_new_label%" button below to open the Carrier Wizard, which will help you register a new carrier in a few steps.'|trans({'%add_new_label%': 'Add new carrier'|trans({}, 'Admin.Shipping.Feature')}, 'Admin.Shipping.Help') }}</p><br>
+    <p>{{ 'Note: DO NOT register a new carrier if there already exists a module for it! Using a module will be much faster and more accurate!'|trans }}</p>
```

Comment:
It is way too long for an information notification, perhaps we should consider detailing it in the user documentation to shorten this notification? It would be more user friendly and looks easier to the user when arriving on this page. What do you think, @Julievrz?

Reply:
Here is how it looks like currently:

<img width="1208" alt="Capture d’écran 2020-09-10 à 12 52 27" src="https://user-images.githubusercontent.com/32565890/92720185-8b71ed80-f364-11ea-85ec-c7ac8cdafd2e.png">

Reply:
It is true that this notification is very long. I agree with you that we should shorten it. The user documentation is actually quite detailed already. So I tried to shorten it like this: 

"Your online store needs to have a proper carrier registered in PrestaShop as soon as you start shipping your products. 
PrestaShop comes with a number of free carrier modules that you can activate. You can also buy carrier modules on the PrestaShop Addons marketplace.
If there is no existing module for your carrier, you can register that carrier manually by clicking on the "Add new carrier". For greater accuracy and speed, please DO NOT register a new carrier if a module already exists for it."

What do you think? @LouiseBonnard

Reply:
Great if the documentation is detailed enough, and nice rewording - I tried to truncate it again, here is my suggestion:

> You need to register all the carriers with which you have chosen to work. PrestaShop comes with a selection of carrier modules you can install right below the page or buy on [1]Addons marketplace[/1].
>
> If there is no existing module for your carrier, you can register it manually by clicking on "[1]" but before, please make sure you did not register a new carrier if a module already exists for it!

[1]Addons marketplace[/1] = https://addons.prestashop.com/en/520-shipping-carriers
[1] = _Add new carrier_ in _Admin.Shipping.Feature_ (cf. https://github.com/PrestaShop/PrestaShop/issues/18809 for all the context)

I think you can drastically shorten this paragraph by keeping only crucial information: we now rather talk about 'store' instead of 'online store', the merchant actually needs to add carriers during all his merchant life so we can remove the 'start shipping your products', we usually avoid the use of 'free' not to confuse the user (the module itself might be free but not the service), prefer 'install' following the terminology used on this page (cf. screenshot below), when possible add information on where it can be installed/found/etc. with the help of links or paths to the proper page of the software, and finally keep the formulation as positive as possible.

<img width="1555" alt="Capture d’écran 2020-09-11 à 15 40 48" src="https://user-images.githubusercontent.com/32565890/92935236-d06f5000-f448-11ea-8fff-fe1602dd7f52.png">

Reply:
Thank you for your advice @LouiseBonnard, I think we can stick with this wording ! 👍

Reply:
😉

Reply:
@jolelievre